modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -25,16 +25,27 @@ You can deploy {lvms} on {sno} clusters using one of the following:
 [id="lvms-deployment-requirements-for-sno-ran_{context}"]
 == Requirements
 
-Before you begin deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
+Before deploying {lvms} on {sno} clusters, ensure that the following requirements are met:
 
 * You have installed {rh-rhacm-first} on an {product-title} cluster.
```

Comment:
In time, is this really a strong requirement or a recommendation? 
I do understand the use case when we are managing SNO deployments with ACM, but it is also feasible to install LVMS on SNO directly without ACM.